index.js

```diff
@@ -5,7 +5,7 @@ var propIsEnumerable = Object.prototype.propertyIsEnumerable;
 
 function toObject(val) {
 	if (val === null || val === undefined) {
-		throw new TypeError('Sources cannot be null or undefined');
+		throw new TypeError('Target cannot be null or undefined');
```

Comment:
@connec What do you think about:
```js
throw new TypeError('Cannot convert undefined or null to object');
```
This would be consistent with `Object.assign`:
```js
> Object.assign(null, {})
TypeError: Cannot convert undefined or null to object
…
```